templates/gbdb.conf.erb

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```

Comment:
Is it possible for the script to read username/password data from the gutterball.conf file? Just trying to reduce the proliferation of password values where possible.